src/bindings/python/flux/cli/base.py

```diff
@@ -1232,6 +1232,11 @@ def init_jobspec(self, args):
         if not args.command:
```

Comment:
Commit message: s/bart/part/

Any issue to reference here?

Reply:
Yeah, I'll just mark it as fixes #6228, since it at least gives users a workaround